.github/workflows/serenity-template.yml

```diff
@@ -140,6 +140,7 @@ jobs:
           CCACHE_DIR: ${{ env.SERENITY_CCACHE_DIR }}
 
       - name: Save Caches
+        if: ${{ inputs.coverage == 'OFF' }}
```

Comment:
Maybe also for LagomGCC? Or is that cache not as large?

Reply:
Cache is probably smaller (~400MiB), and it would first need to pass in order to generate a cache :eyes: